tools/importer-rest-api-specs/cmd/segments.go

```diff
@@ -50,7 +50,7 @@ func (c SegmentsCommand) Run(_ []string) int {
 		DataApiEndpoint:          nil,
 		JustOutputSegments:       true,
 		Logger:                   logger,
-		OutputDirectory:          os.DevNull,
+		OutputDirectoryCS:        os.DevNull,
```

Comment:
Do we need `/dev/null/` in here for the yaml path for the point we remove the CS?